app/helpers/account_activations_helper.rb

```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```

Comment:
Please remove such modules if you don't need them

Reply:
fixed in PR#9